ils_middleware/tasks/amazon/sqs.py

```diff
@@ -27,12 +27,12 @@ def SubscribeOperator(**kwargs) -> SQSSensor:
     )
 
 
-def get_group(resource_uri: str) -> str:
-    """Retrieves the Resource's Group."""
+def get_resource(resource_uri: str) -> dict:
+    """Retrieves the Resource"""
     result = requests.get(resource_uri)
     if result.status_code < 400:
-        return result.json().get("group")
-    return f"{resource_uri} returned error {result.status_code}"
+        return result.json()
+    raise ValueError(f"{resource_uri} returned error {result.status_code}")
```

Comment:
It looks like the error message here isn't surfaced (because it appears to be being eaten on line 50). Should it be logged? If not, I wonder if instead of raising for control flow this line could return `{}`? That way the try/except below could go away altogether. Don't feel strongly about aaaaaaaaaany of this. 🤷🏻‍♂️

Reply:
That would help the small coverage drop....